modules/standard/Random.chpl

```diff
@@ -399,7 +403,7 @@ module Random {
             var (found, indexChosen) = Search.binarySearch(cumulativeArr, randNum);
             if !indicesChosen.contains(indexChosen) {
               indicesChosen += indexChosen;
-              samples[i] = A[indexChosen];
+              samples[i] = (indexChosen-1) * stride + low;
```

Comment:
I wonder if you could use [`range.orderToIndex`](https://chapel-lang.org/docs/master/builtins/ChapelRange.html#ChapelRange.range.orderToIndex) here?

Reply:
Yes, same goes for here as well :-)